nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -551,11 +548,35 @@ def _get_metrics_calculator(self) -> MetricsCalculator:
 
 
 class ActivationAPoZRankPruner(ActivationPruner):
+    def _collector(self, buffer: List) -> Callable[[Module, Tensor, Tensor], None]:
+        assert len(buffer) == 0, 'Buffer pass to activation pruner collector is not empty.'
```

Comment:
Better to add comment here: what are buffer[0] and buffer[1].

Reply:
added